python/pminit/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "pminit"
+version = "0"
+description = "Post-install hook for PythonMonkey"
+authors = [
+  "Tom Tang <xmader@distributive.network>",
+  "Caleb Aikens <caleb@distributive.network>",
+  "Wes Garland <wes@distributive.network>",
+  "Hamada Gasmallah <hamada@distributive.network>"
+]
+include = [
+  # Install extra files into the pythonmonkey package
+  "pythonmonkey/package*.json",
+  { path = "pythonmonkey/node_modules/**/*", format = "wheel" },
+  # pip builds (and actually installs from) the wheel when installing from sdist
+  # we don't want node_modules inside the sdist package
+]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+
+[tool.poetry.build]
+script = "post-install-hook.py"
```

Comment:
This command just typically install `post-install-hook` as a cli script that can latter be callable. I don't believe this will actually add the setup.py post install hook unfortunately.... 

My understanding is that the only way to get a post install hook is to use `setup.py` instead of `pyproject.toml` and then doing something like what [this post recommends](https://stackoverflow.com/questions/20288711/post-install-script-with-python-setuptools). 

Have you tried the generated source dist? Does it work?

Reply:
Seeing your actions, it seems like it works, do you have a link to the docs for this type of workflow? I'm just interested to see where they documented it.

Reply:
`post-install-hook.py` is not actually a post-install hook for setup.py you thought of.

`post-install-hook.py` itself is the setup script (like `setup.py`) Poetry will use - You can name it to anything.